src/Elastic.Documentation.Site/Assets/web-components/AppliesToPopover.tsx

```diff
@@ -0,0 +1,494 @@
+                onMouseEnter: handleMouseEnter,
+                onMouseLeave: handleMouseLeave,
+                css: css`
+                    max-width: 420px;
+                    z-index: 40 !important; /* Show below the main header if needed */
+
+                    /* Remove all focus styling from panel */
+                    &,
+                    &:focus,
+                    &:focus-visible,
+                    &:focus-within {
+                        outline: none !important;
+                        box-shadow: 0 4px 16px rgba(0, 0, 0, 0.1) !important;
+                    }
+
+                    .euiPopover__panelArrow {
+                        display: none;
+                    }
+                `,
+            }}
+        >
+            <div
+                ref={contentRef}
+                css={css`
+                    padding: 16px;
+                    font-size: 14px;
+                    line-height: 1.5;
+                    color: var(--color-grey-100, #1a1c21);
+                `}
+            >
+                {/* Product description */}
+                {popoverData?.productDescription && (
+                    <p
+                        css={css`
+                            margin: 0 0 16px 0;
+                            color: var(--color-grey-80, #343741);
+
+                            strong {
+                                display: inline;
+                                font-weight: 700;
+                                color: var(--color-grey-100, #1a1c21);
+                            }
+                        `}
+                        dangerouslySetInnerHTML={{
+                            __html: popoverData.productDescription,
+                        }}
+                    />
+                )}
+
+                {/* Availability section */}
+                {popoverData && popoverData.availabilityItems.length > 0 && (
+                    <>
+                        <p
+                            css={css`
+                                margin: 0 0 4px 0;
+                            `}
+                        >
+                            <strong
+                                css={css`
+                                    display: inline;
+                                    font-weight: 700;
+                                    font-size: 15px;
+                                    color: var(--color-grey-100, #1a1c21);
+                                `}
+                            >
+                                Availability
+                            </strong>
+                        </p>
+                        <p
+                            css={css`
+                                margin: 0 0 8px 0;
+                                color: var(--color-grey-70, #535966);
+                                font-size: 13px;
+                            `}
+                        >
+                            The functionality described here is:
```

Comment:
I think we could remove this line @elastic/docs-tech-leads @vishaangelova, do you think it's very valuable?